dev/com.ibm.ws.jsfContainer_fat_2.3/publish/files/MANIFEST_badMojarra_40.MF

```diff
@@ -0,0 +1,290 @@
+ faces.core;version="[4.0,5)",com.sun.faces.facelets.tag.faces.html;vers
+ ion="[4.0,5)",com.sun.faces.facelets.tag.jstl.core;version="[4.0,5)",co
+ m.sun.faces.facelets.tag.jstl.fn;version="[4.0,5)",com.sun.faces.facele
+ ts.tag.ui;version="[4.0,5)",com.sun.faces.facelets.util;version="[4.0,5
+ )",com.sun.faces.flow;version="[4.0,5)",com.sun.faces.flow.builder;vers
+ ion="[4.0,5)",com.sun.faces.io;version="[4.0,5)",com.sun.faces.lifecycl
+ e;version="[4.0,5)",com.sun.faces.push;version="[4.0,5)",com.sun.faces.
+ renderkit;version="[4.0,5)",com.sun.faces.renderkit.html_basic;version=
+ "[4.0,5)",com.sun.faces.spi;version="[4.0,5)",com.sun.faces.util;versio
+ n="[4.0,5)",com.sun.faces.util.copier;version="[4.0,5)",com.sun.faces.v
+ endor;version="[4.0,5)",jakarta.servlet.annotation;resolution:=optional
+ ;version="[6.0,7)",jakarta.servlet;version="[6.0,7)",jakarta.servlet.ht
+ tp;version="[6.0,7)",jakarta.el;version="[5.0,6)",jakarta.inject;versio
+ n="[2.0,3)",jakarta.enterprise.inject;version="[4.0,5)",jakarta.enterpr
+ ise.inject.spi;version="[4.0,5)",jakarta.enterprise.util;version="[4.0,
+ 5)",jakarta.enterprise.context;version="[4.0,5)",jakarta.enterprise.con
+ text.spi;version="[4.0,5)",jakarta.annotation;version="[2.1,3)",javax.c
+ rypto,javax.crypto.spec,jakarta.websocket;resolution:=optional;version=
+ "[2.1,3)",jakarta.websocket.server;resolution:=optional;version="[2.1,3
+ )",jakarta.json;resolution:=optional;version="[2.1,3)",jakarta.json.str
+ eam;resolution:=optional;version="[2.1,3)",jakarta.validation;resolutio
+ n:=optional;version="[3.0,4)",jakarta.validation.groups;resolution:=opt
+ ional;version="[3.0,4)",jakarta.enterprise.event;resolution:=optional;v
+ ersion="[4.0,5)",jakarta.ejb;resolution:=optional;version="[4.0,5)",jak
+ arta.persistence;resolution:=optional;version="[3.1,4)",javax.xml.names
+ pace,javax.xml.parsers,javax.xml.transform,javax.xml.transform.dom,java
+ x.xml.transform.stream,javax.xml.validation,javax.xml.xpath,org.w3c.dom
+ ,org.w3c.dom.ls,org.xml.sax,org.xml.sax.ext,org.xml.sax.helpers,javax.n
+ aming,javax.naming.spi
+Originally-Created-By: Apache Maven Bundle Plugin 5.1.4
+Require-Capability: osgi.serviceloader;filter:="(osgi.serviceloader=com.
+ sun.faces.spi.AnnotationProvider)";resolution:=optional;cardinality:=mu
+ ltiple,osgi.serviceloader;filter:="(osgi.serviceloader=com.sun.faces.sp
+ i.FaceletConfigResourceProvider)";resolution:=optional;cardinality:=mul
+ tiple,osgi.serviceloader;filter:="(osgi.serviceloader=com.sun.faces.spi
+ .FacesConfigResourceProvider)";resolution:=optional;cardinality:=multip
+ le,osgi.serviceloader;filter:="(osgi.serviceloader=jakarta.faces.applic
+ ation.ApplicationConfigurationPopulator)";resolution:=optional;cardinal
+ ity:=multiple,osgi.extender;filter:="(osgi.extender=osgi.serviceloader.
+ processor)";resolution:=optional,osgi.ee;filter:="(&(osgi.ee=JavaSE)(ve
+ rsion=11))"
+Specification-Title: Jakarta Faces
+Specification-Version: 3.0
```

Comment:
Test looks at this `Specification-Version` which is 3.0 here when it should be 4.0.

Test expects a failure.